posts/2024-06-21-june-meetup/index.qmd

```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title:  "SheffieldR : June 2024 Meetup"
+date:   2024-06-21 16:00:00 +0000
+categories: meetups
+image: image.png
+excerpt_separator: <!--more-->
+---
+
+This month, we're shaking things up with an exciting line-up of lightning talks presented by you, our vibrant community members! This is your chance to share your knowledge, showcase your projects, and discuss your favourite R topics in a quick, 5-10 minute format.
+
+If you have a fascinating R project you're working on, a useful package you can't live without, or a data analysis technique you think others should know about sign up to present a lightning talk and share it with the SheffieldR community!
+
+Whether you're a seasoned R user or just starting out, we encourage everyone to participate. This is a fantastic opportunity to learn from each other, exchange ideas, and inspire fellow R enthusiasts.
+
+![](image.png)
+
+How to Participate:
+
+- Register Your Topic: Fill out [this form](https://forms.gle/YSWXarhwUpXZQp2J6) to register your lightning talk topic.
+
+- Prepare Your Talk: Visual aids are welcome but not required.
+
+- Engage and Enjoy: Join us on the day, share your insights, and enjoy learning from others.
+
+After the event, join us for drinks at the The University Arms pub, it's a great opportunity to continue the conversation, network, and relax with fellow R enthusiasts.
```

Comment:
Adding link to University Arms on OpenStreetMap to help people find it (although I expect most to wander from Hicks straight there but you never know some later arrivals might just want to head to the pub!).

```suggestion
After the event, join us for drinks at the [The University Arms pub](https://www.openstreetmap.org/way/178278884), it's
a great opportunity to continue the conversation, network, and relax with fellow R enthusiasts.
```